src/core/server/saved_objects/service/lib/search_dsl/query_params.ts

```diff
@@ -224,6 +246,17 @@ export function getQueryParams({
     ],
   };
 
+  if (workspaces) {
```

Comment:
Do we need to check the length of workspaces != 0?

Reply:
Sure, and we should check if the array contains undefined or empty string, done for this.